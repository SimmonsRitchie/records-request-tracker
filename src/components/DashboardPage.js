import React from 'react';
import RequestsListFilters from './RequestsListFilters'
import RequestsList from './RequestsList'
import DashboardSummary from './DashboardSummary';


const DashboardPage = () => (
    <div>
        <DashboardSummary />
        <RequestsListFilters />
        <RequestsList />
    </div>
);

export default DashboardPage;