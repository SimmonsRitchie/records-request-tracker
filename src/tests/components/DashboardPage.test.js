import React from 'react';
import { shallow } from 'enzyme';
import DashboardPage from '../../components/DashboardPage'

test('Should render DashboardPage with expenses', () => {
    const wrapper = shallow(<DashboardPage />)
    expect(wrapper).toMatchSnapshot();
});