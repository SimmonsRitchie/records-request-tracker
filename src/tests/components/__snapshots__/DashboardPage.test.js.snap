// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render DashboardPage with expenses 1`] = `
<div>
  <Connect(DashboardSummary) />
  <Connect(RequestsListFilters) />
  <Connect(FiltersSummary) />
  <Connect(RequestList) />
</div>
`;
